packages/babel/src/transformation/modules/common.js

```diff
@@ -91,8 +92,11 @@ export default class CommonJSFormatter extends DefaultFormatter {
           t.variableDeclarator(variableName, ref)
         ]));
       } else {
+        var importedIdentifier = t.cloneDeep(specifier.imported);
+        traversal.clearNode(importedIdentifier);
```

Comment:
Oh yeah, I guess I can do that too. I like that better. Do you think I should still add the comment with the issue number in that case?